odbc-api/src/handles/environment.rs

```diff
@@ -175,7 +175,7 @@ impl Environment {
     ///
     /// [1]: https://docs.microsoft.com/sql/odbc/reference/syntax/sqldrivers-function
     pub fn drivers_buffer_len(
-        &mut self,
```

Comment:
Sorry, I missed this the first time around. I do not think (hope?) this is unsafe, as some synchronization is done by the driver manager. Yet I wonder if there should be at least some comment explaining why it is safe to be shared / immutable

Reply:
Sure, I can clarify it in a comment here